src/VisualStudio/Core/Def/Implementation/LanguageService/AbstractPackage.cs

```diff
@@ -4,14 +4,34 @@
 
 #nullable disable
 
+using System;
 using System.Threading;
+using Microsoft.VisualStudio.ComponentModelHost;
 using Microsoft.VisualStudio.Shell;
 using Task = System.Threading.Tasks.Task;
 
 namespace Microsoft.VisualStudio.LanguageServices.Implementation.LanguageService
 {
     internal abstract class AbstractPackage : AsyncPackage
     {
+        private IComponentModel _componentModel_doNotAccessDirectly;
```

Comment:
💭 Not a huge fan of `_doNotAccessDirectly` here... but hopefully this can be removed once we can have partial-auto-properties.